### PR TITLE
[KARAF-7754] Upgrade to Pax Web 9.0.11 & Jetty 10.0.16

### PR DESCRIPTION
**Pull request by rovarga**
Signed-off-by: Robert Varga <robert.varga@pantheon.tech>

### CODE REVIEW

**Comment by grgrzybek** [comment]
Please check https://issues.apache.org/jira/browse/KARAF-7544 and my previous attempt: https://github.com/grgrzybek/karaf/commits/KARAF-7544-Jetty10-PaxWeb9

File: examples/karaf-websocket-example/src/main/java/org/apache/karaf/examples/websocket/WebsocketExampleServlet.java [hunk 1]
```diff
@@ -16,16 +16,16 @@
...
-import org.eclipse.jetty.websocket.servlet.WebSocketServlet;
-import org.eclipse.jetty.websocket.servlet.WebSocketServletFactory;
+import org.eclipse.jetty.websocket.server.JettyWebSocketServlet;
+import org.eclipse.jetty.websocket.server.JettyWebSocketServletFactory;
 
 import javax.servlet.annotation.WebServlet;
 
 @WebServlet(name = "Example WebSocket Servlet", urlPatterns = { "/example-websocket "})
-public class WebsocketExampleServlet extends WebSocketServlet {
+public class WebsocketExampleServlet extends JettyWebSocketServlet {
```
Comment (jbonofre) [inline examples/karaf-websocket-example/src/main/java/org/apache/karaf/examples/websocket/WebsocketExampleServlet.java:25]: We should use the websocket handling here, not the Jetty one.

Comment (rovarga) [reply examples/karaf-websocket-example/src/main/java/org/apache/karaf/examples/websocket/WebsocketExampleServlet.java:25]: > We should use the websocket handling here, not the Jetty one.

This is just the Jetty 9.4  -> 10 renaming. Do you mean to switch the example to just use JSR356?

**Comment by rovarga** [comment]
> Please check https://issues.apache.org/jira/browse/KARAF-7544 and my previous attempt: https://github.com/grgrzybek/karaf/commits/KARAF-7544-Jetty10-PaxWeb9

Argh, sorry for the confusion and thanks for the pointer. I will be closing this PR and re-start with your attempt as it contains quite a few clean ups which I think we can merge before making the jump.

**Comment by jbonofre** [comment]
Better to resume what @grgrzybek started imho.

**Comment by grgrzybek** [comment]
I'll get back to https://github.com/grgrzybek/karaf/commits/KARAF-7544-Jetty10-PaxWeb9 at some point.

I'm working on Jolokia now and I need Pax Web with jakarta support, so I got back to Pax Web 10 and realized that Pax URL War needs `jakarta.servlet` version too, so I started checking Pax URL 3 (work in progress) and realized there are Maven Resolver things to solve (we should move to Maven Resolver 2 soon).

But I didn't lost track of what I work on ;)

(https://www.youtube.com/watch?v=AbSehcT19u0)

**Pull request closed without merging**